appveyor.yml

```diff
@@ -102,6 +102,7 @@ for:
   artifacts:
     # upload the generated portable archive only
     - path: 'Setup/GitExtensions-Portable-*.zip'
+    - path: 'Setup/GitExtensions-*.msi'
```

Comment:
We have not uploaded .msi previously, I believe that the reason was that it makes it too easy to use unstable builds. Also, normal builds are not signed.
For testing the PR before merging it is of course OK

Reply:
What part of "Temp commit to verify installer build" do you not understand?

Reply:
Jay, we are just making notes for ourselves. No need to get angry please.